docs/reference/how-to/search-speed.asciidoc

```diff
@@ -10,6 +10,7 @@ goes to the filesystem cache so that Elasticsearch can keep hot regions of the
 index in physical memory.
 
 [discrete]
+tag::readahead[]
```

Comment:
```suggestion
tag::readahead[]
[discrete]
```
Is "Avoid page cache thrashing by using modest readahead values on Linux" broken out into its own page on purpose? If not, swap these two lines.

Reply:
Oops, thanks!